internal/pkg/service/cli/cmd/ci/workflow/workflows_test.go

```diff
@@ -1,22 +1,36 @@
-package dialog_test
+package workflow
 
 import (
 	"sync"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 
-	"github.com/keboola/keboola-as-code/internal/pkg/service/cli/cmd/ci/workflow"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/cli"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/cli/dialog"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/cli/options"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/common/configmap"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/testhelper/terminal"
 	genWorkflows "github.com/keboola/keboola-as-code/pkg/lib/operation/project/local/workflows/generate"
 )
 
 func TestAskWorkflowsOptionsInteractive(t *testing.T) {
 	t.Parallel()
 
-	dialog, _, console := createDialogs(t, true)
+	// options
+	o := options.New()
 
-	f := workflow.DefaultFlags()
+	// terminal
+	console, err := terminal.New(t)
+	require.NoError(t, err)
+
+	p := cli.NewPrompt(console.Tty(), console.Tty(), console.Tty(), false)
+
+	// dialog
+	d := dialog.New(p, o)
```

Comment:
Please, replace this duplicate code with a common function, for example `dialog.NewForTest(...)`.

Reply:
@hosekpeter Is this fixed? 

![image](https://github.com/keboola/keboola-as-code/assets/19371734/51a3fa30-1906-4e63-85f7-dd7e770a7f1e)

Reply:
@michaljurecko yes -> https://github.com/keboola/keboola-as-code/pull/1626/commits/fa9432750726a8f54da6e2c82434a07f2bf49f0a

Reply:
:man_shrugging: And also `TestAskWorkflowsOptionsInteractive` function?

![image](https://github.com/keboola/keboola-as-code/assets/19371734/c2143793-e3ef-420f-ac2e-9750cbc06427)

Reply:
done